library/src/main/java/com/northconcepts/templatemaster/form/presenter/NumberPresenter.java

```diff
@@ -0,0 +1,36 @@
+package com.northconcepts.templatemaster.form.presenter;
+
+import java.text.DecimalFormat;
+
+import com.northconcepts.templatemaster.form.CrudResource;
+import com.northconcepts.templatemaster.form.FieldDef;
+
+public class NumberPresenter implements FieldValuePresenter {
+
+    public static final NumberPresenter DOLLARS_NO_CENTS = new NumberPresenter("$#,##0");
+    public static final NumberPresenter DOLLARS_CENTS = new NumberPresenter("$#,##0.00");
+    public static final NumberPresenter WHOLE_NO_GROUPING = new NumberPresenter("0");
+    public static final NumberPresenter WHOLE_GROUPING = new NumberPresenter("#,##0");
+    public static final NumberPresenter DECIMAL_NO_GROUPING = new NumberPresenter("0.#");
+    public static final NumberPresenter DECIMAL_GROUPING = new NumberPresenter("#,##0.#");
+
+    private DecimalFormat decimalFormat;
+
+    public NumberPresenter(String decimalFormat) {
```

Comment:
It seems we've removed the overloaded constructor that accepts a DecimalFormat instance.  Please add it back.

Reply:
I thought we agreed to remove that constructor. But that's fine. I am adding this constructor for both NumberPreseneter & NumberEditor.